tox.ini

```diff
@@ -35,6 +35,14 @@ deps = -r{toxinidir}/requirements.txt
 commands =
   twine check dist/*
 
+[testenv:mypy]
+basepython = python3
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+       mypy
+commands =
+  mypy {posargs} gitlab/base.py gitlab/client.py gitlab/cli.py gitlab/config.py gitlab/const.py gitlab/exceptions.py gitlab/__init__.py gitlab/__main__.py gitlab/mixins.py gitlab/types.py gitlab/utils.py gitlab/__version__.py
```

Comment:
I guess this list will grow as more typing is added.

Can we split this into multiple lines or maybe use `setup.cfg`/`mypy.ini` if it helps readability? I see you can define patterns as well, if that helps. https://mypy.readthedocs.io/en/stable/config_file.html

Reply:
I will read up on that. Thanks.

Reply:
Added a `.mypy.ini` and there I was able to use a wildcard of `gitlab/*.py`